src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Review.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@using SFA.DAS.Encoding
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort.ReviewViewModel
+@inject IEncodingService EncodingService
```

Comment:
As on Draft page, we could encode in the view model mapper

Reply:
Oops, I moved the code to mapper, but forgot to remove the injection, nice spot thanks.